model/tensorflow_hub/dffml_model_tensorflow_hub/text_classifier.py

```diff
@@ -0,0 +1,364 @@
+# SPDX-License-Identifier: MIT
+# Copyright (c) 2019 Intel Corporation
+"""
+Description of what this model does
+"""
+import abc
+import inspect
+from typing import AsyncIterator, Tuple, Any, List, Optional, NamedTuple, Type
+import os
+import hashlib
+
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
+from dffml.repo import Repo
+from dffml.source.source import Sources
+from dffml.feature import Features
+from dffml.accuracy import Accuracy
+from dffml.model.model import ModelConfig, ModelContext, Model
+from dffml.util.entrypoint import entrypoint
+from dffml.base import BaseConfig, config, field
+from dffml.feature.feature import Feature, Features
+from dffml_model_tensorflow.dnnc import TensorflowModelContext
+import tensorflow as tf
```

Comment:
Since we're on the topic, make sure to follow https://github.com/intel/dffml/issues/339 (so tensorflow, numpy, etc. before dffml)